modules/k8s/version.go

```diff
@@ -0,0 +1,18 @@
+package k8s
+
+import "github.com/gruntwork-io/terratest/modules/testing"
+
+// GetKubernetesClusterVersion returns the Kubernetes cluster version.
+func GetKubernetesClusterVersion(t testing.TestingT) (string, error) {
```

Comment:
NIT: We label functions that return errors with a `E`

```suggestion
func GetKubernetesClusterVersionE(t testing.TestingT) (string, error) {
```

Reply:
Totally agree. I'm having trouble figuring out how to unit test this function since it depends on [Kubernetes client](https://github.com/gruntwork-io/terratest/blob/adca4a84bcddefbb7d6453fb8bd17da25f2ecd04/modules/k8s/client.go#L16). 

I initially thought of accepting the client as input to the function to allow for passing a mock client for testing. However, this raises the complexity of the function that should be an easy to call helper function.

Got an idea on how to test this?

Reply:
Two thoughts:

- It might be a good idea to introduce an additional function, `GetKubernetesClusterVersionWithOptionsE(t testing.TestingT, kubectlOptions *KubectlOptions) (string, error)`, and use `GetKubernetesClientFromOptionsE` instead of `GetKubernetesClient`. This way, you can have full control over how to authenticate by constructing custom KubectlOptions. If you do do this, `GetKubernetesClusterVersionE` should call `GetKubernetesClientFromOptionsE`, constructing the options using the same way as https://github.com/gruntwork-io/terratest/blob/master/modules/k8s/client.go#L16

- You can actually write the tests ignoring injection, and rely on the default context against minikube. This is actually how most of the existing tests for the module are written.


I have a slight preference for the first option, but will accept this PR with either approach!

Reply:
Implemented suggestion number 1 and added unit tests.